content/blog/intro-to-html-css-and-savascript/index.md

```diff
@@ -0,0 +1,71 @@
+---
+{
+    title: "Introduction to HTML, CSS, and JavaScript",
```

Comment:
```suggestion
	title: "Introduction to HTML, CSS, and JavaScript",
```

- Switched to tabs to match the style of most of the others, same as the next few suggestions (I will create an issue for the rest of the posts that aren't)